cloud_Azure/terraform/module/README.md

```diff
@@ -1,12 +1,21 @@
 # Azure Kentik integration Terraform module
 
-Terraform module which creates Azure resources required for Kentik to enable integration
+Terraform module, which creates Azure and Kentik resources, for Azure cloud integration in Kentik.  
```

Comment:
"Module supporting management of Azure and Kentik resources required for flow log export from Azure to Kentik.